File: Data Structures & Algorithms/two-integer-sum-ii/submission-2.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int n = numbers.length;


        int L = 0;
        int R = n - 1;
        int[] res = new int[2];

        while (L < R) {
            if (numbers[L] + numbers[R] > target) {
                R--;
            } else if (numbers[L] + numbers[R] < target) {
                L++;
            } else {
                res[0] = L + 1;
                res[1] = R + 1;
                return res;
            }
        }

        return res;
    }

}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        mp = {}


        for i in range(len(nums)):
            if nums[i] not in mp:
                mp[nums[i]] = 1
            else: 
                mp[nums[i]] += 1
        

        res = []
        while k > 0:
            greatest_val = 0
            kth_greatest_elem = 0 
            for key in mp:
                if mp[key] > greatest_val:
                    kth_greatest_elem = key
                    greatest_val = mp[key]

            res.append(kth_greatest_elem)
            mp.pop(kth_greatest_elem)
            k -= 1


        return res